### DISCUSSION TITLE
PasswordPusher in a Rasperry Pi4

### DISCUSSION THREAD

**Discussion by bizkeliza** [pglombardo/PasswordPusher#2644] [General]
Greetings to the community.
I would like to know if it is possible to install PasswordPusher on a Rasperry. 
Thank you

**Comment by pglombardo** [comment]
Cool idea.  To answer your question, I've never seen it done but it should work with some tweaks.

For sure I would set the environment variable `WEB_CONCURRECY` to 1 or 2 so it doesn't launch too many processes and exhausts memory.

From Source might be troublesome but if the Raspberry OS can run Ruby, Node & SQLite3 I believe it would work.  Those are the only real requirements.

Easier if you install Docker on your raspberry pi, you can just run the container.  4GB memory should be enough.

**Comment by bizkeliza** [comment]
My knowledge is not so extensive that I can do it. In the end I am going to install it on an Ubuntu, is there a manual from which I can follow the installation step by step? Thank you

**Reply by pglombardo** [reply]
The best and easiest way is to run the Docker container.  I advise against the installing with source.  There are too many steps and it's hard for me to support.

There are some install docs here:
https://docs.pwpush.com/docs/installation/
